private_sharing/forms.py

```diff
@@ -73,6 +73,30 @@ def __init__(self, *args, **kwargs):
             # coerce the result to a boolean
             self.fields[field].coerce = lambda x: x == 'True'
 
+    def clean(self):
+        """
+        Logic to for conditional required elements in our form.
+        """
+        cleaned_data = super().clean()
+
+        add_data = cleaned_data.get('add_data', False)
+        explore_share = cleaned_data.get('explore_share', False)
+        returned_data_description = cleaned_data.get(
+            'returned_data_description', None)
+
+        if not add_data or explore_share:
+            self.add_error('add_data',
+                           forms.ValidationError('Please pick "Add data" or '
```

Comment:
I think this error message isn't fully correct. People can assign their projects to be both, 'add data' and 'explore & share'. So it should be re-written to 'Pick at least one option from 'add data' and 'explore & share'" or something along the lines.

Reply:
Done